### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'json' Possible source of error: frappe_whatsapp (app)

### DISCUSSION THREAD

**Discussion by aqilmujahidiin** [shridarpatil/frappe_whatsapp#55] [Q&A]
hello, I am a frappe_whatsapp user and this is a very good application, here I get an error when clicking the trigger to send a WhatsApp message, which appears as in the following image

used for sending message when doc.status= apporved

output error as below:

AttributeError: 'NoneType' object has no attribute 'json'
Possible source of error: frappe_whatsapp (app)

![Screenshot 2024-01-26 081447](https://github.com/shridarpatil/frappe_whatsapp/assets/25211316/d87f669d-bc9c-4414-aca4-24c66a7b9d57)

**Comment by aqilmujahidiin** [comment]
the problem is when i included date

![Screenshot 2024-01-26 112331](https://github.com/shridarpatil/frappe_whatsapp/assets/25211316/7606f12f-cfa5-4dff-9c33-98679628135d)


when i removed it, no error

![Screenshot 2024-01-26 112348](https://github.com/shridarpatil/frappe_whatsapp/assets/25211316/3b223940-368b-4720-8589-08166082927e)

**Comment by medsun88** [comment]
For the error 
Migrate the site using the below command and check after that 

bench --site [sitename] migrate

**Reply by aryannitesh** [reply]
Not working after migrating
